### ISSUE TITLE
Problemas com valores negativos na venda

### ISSUE THREAD

**Issue by rodrigogpontes** [RuanRCA/ProjetoMercado#1]
Olá. Encontrei problemas ao tentar realizar vendas de produtos utilizando seu sistema.

Ao inserir um valor negativo, o sistema aceita remover o valor total. Creio que o comportamento adequado seja não permitir valores negativos para este campo.

Agradeço desde já a atenção.

**Comment by llen88** [comment]
Concordo, o projeto falta muita estrutura. Não é muito eficiente porém é funcional e me serviu, desejo sorte com o polimento do seu código, SUCESSO.

**Comment by kaio760** [comment]
Achei esse seu projeto uma coisa de iniciante, deveria melhorar, fica a dica
